source/docs/articles/organizations/saml-for-orgs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: SAML for Orgs

```

Comment:
How about something a little more descriptive? Maybe "Single sign-on for Pantheon Organizations"